### ISSUE TITLE
improve ux

### ISSUE THREAD

**Issue by rajvinder-kaur** [OmkarKirpan/GuessColors#9]
this project can be improved the ux of the project for users

**Comment by OmkarKirpan** [comment]
Hi @rajvinder-kaur, 
Thanks for your feedback. We're committed to improving the UX of our color-guessing game. 
Could you please provide more details or specific suggestions on what aspects of the UX you think could be improved? Your input is valuable in helping us make the game better.

**Comment by rajvinder-kaur** [comment]
Can I work on this issue ? And then u can review it and we can improve it together by mutual discussion

**Comment by OmkarKirpan** [comment]
Certainly! We appreciate your interest in contributing to the project. 
Feel free to work on the issue and submit a pull request once you're done. We can then review it together and discuss any potential improvements that can be made.
Please let me know if you need any assistance or have any questions while working on the issue.

**Comment by OmkarKirpan** [comment]
Looking forward to collaborating with you on enhancing the user experience of our color guessing game!

**Comment by rajvinder-kaur** [comment]
hey i have opened a pull request can u please review it
